### PR TITLE
feat(deploy): v2.2.0 changes

### PR DESCRIPTION
**Pull request by niladrih**
- Chart.yaml version, appVersion update
- Chart.yaml openebs website URL update
- README.md image tag, k8s sidecar container registry update
- values.yaml release version, csi plugin image tag, k8s sidecar container registry update

### CODE REVIEW

**Review by avishnu — APPROVED** [review]
lgtm

File: deploy/zfs-operator.yaml [hunk 1]
```diff
@@ -2058,7 +2058,7 @@ spec:
       serviceAccount: openebs-zfs-controller-sa
       containers:
         - name: csi-resizer
-          image: k8s.gcr.io/sig-storage/csi-resizer:v1.2.0
```
Comment (pawanpraka1) [inline deploy/zfs-operator.yaml:2061]: deploy/zfs-operator.yaml is autogenerated yaml. This change will go away once you do make manifests

**Comment by pawanpraka1** [comment]
@niladrih please make the changes in deploy/yamls/zfs-driver.yaml and run the make manifest commnad to autogenerate the deploy/zfs-operator.yaml.

File: deploy/zfs-operator.yaml (hunk 1 above)
Comment (niladrih) [reply deploy/zfs-operator.yaml:2061]: Done, good catch!

**Comment by pawanpraka1** [comment]
looks good. Please update the yaml in the charts repo also.

**Review by pawanpraka1 — APPROVED** [review]

**Pull request merged**